model-archiver/model_archiver/model_packaging.py

```diff
@@ -38,7 +38,9 @@ def package_model(args, manifest):
         # Step 3 : Zip 'em all up
         ModelExportUtils.zip(export_file_path, model_name, model_path, files_to_exclude, manifest)
         logging.info("Successfully exported model %s to file %s", model_name, export_file_path)
-
+    except Exception as e:  # pylint: disable=broad-except
+        logging.error(e)
```

Comment:
For ModelArchiverError, we should not print stacktrace, but other exception, we should.